modules/installation-nutanix-infrastructure.adoc

```diff
@@ -13,6 +13,8 @@ You must install the {product-title} cluster to a Nutanix environment that meets
 [cols=2, options="header"]
 |===
 |Component |Required version
-|Nutanix AOS | 5.20.4+ or 6.5.1+
-|Prism Central | 2022.4+
+|Nutanix AOS | Long Term Support (LTS) release 6.5.2.7 or later
+
```

Comment:
We will support 6.5.2.7 or later, regardless LTS or STS. We should remove all references to LTS or STS and just leave it as "Nutanix AOS release 6.5.2.7 or later", which includes both, and both are supported by Nutanix.